war/src/main/js/components/dropdowns/utils.js

```diff
@@ -86,7 +91,11 @@ function generateDropdownItems(items, compact) {
     menuItems,
     () => menuItems.querySelectorAll(".jenkins-dropdown__item"),
     SELECTED_ITEM_CLASS,
-    (selectedItem, key) => {
+    (selectedItem, key, evt) => {
+      if (!selectedItem) {
+        return;
+      }
```

Comment:
occurs when navigating between potential text left and right without selecting an item (doesn't cause an issue just stacktraces in the console)

e.g. type North, then navigate left on the keyboard